target/arm/translate-a64.c

```diff
@@ -86,6 +86,8 @@ void a64_translate_init(void)
 
     cpu_exclusive_high = tcg_global_mem_new_i64(cpu_env,
         offsetof(CPUARMState, exclusive_high), "exclusive_high");
+
+    qemu_log_printf_create_globals();
```

Comment:
You may also get away with putting this initialization into something like an `std::call_once` in `qemu_log_instr_init()`, not sure whether it is called too early though.

Reply:
Also the jenkins failure seems to be due to a missing `#include` here